applications/vessel-history/src/features/profile/components/activity/ActivityModalContentDetailsPortVisit.tsx

```diff
@@ -75,10 +78,22 @@ const ActivityModalContentDetailsPortVisit: React.FC<ActivityModalContentProps>
         <ActivityModalContentField
           label={t('event.confidence', 'Confidence')}
           value={confidence ?? DEFAULT_EMPTY_VALUE}
+          help={help}
```

Comment:
This is not being used, it's always an empty string so if can be removed here, the [const definition](https://github.com/GlobalFishingWatch/frontend/pull/901/files#diff-f88fe68aee88d173d7f856b95647f54e7237a53d15c20fe78d42444e0f36f47aR61) and also ervert the changes on `applications/vessel-history/src/features/profile/components/activity/ActivityModalContentField.tsx` component to support this new help prop. Makes sense?